package.json

```diff
@@ -56,15 +56,23 @@
   },
   "devDependencies": {
     "babel-eslint": "^8.0.1",
+    "babel-jest": "^21.2.0",
     "babel-plugin-lodash": "^3.2.11",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
```

Comment:
No need of all these devDependencies I think. since we have these in `.babelrc`
https://github.com/coderplex/coderplex/blob/f9d9478af33be8b32cfffaa6b4e246702ddc6da9/.babelrc#L9-L14

Reply:
Ah.. so the magic lies with in next.js 🤔  hm never digged into it anyways

Reply:
Yes

Reply:
@buoyantair yes the code for next/babel is [here](https://github.com/zeit/next.js/blob/canary/server/build/babel/preset.js) 
@vinaypuppal yes i havent had chance to look into next/babel implementation. You can write they bunched those presents into one and packed it with next js.